simpeg_drivers/electromagnetics/frequency_domain/params.py

```diff
@@ -11,68 +11,61 @@
 
 from __future__ import annotations
 
-from copy import deepcopy
-from uuid import UUID
+from pathlib import Path
+from typing import ClassVar, TypeAlias
+
+from geoh5py.groups import PropertyGroup
+from geoh5py.objects import (
+    AirborneFEMReceivers,
+    LargeLoopGroundFEMReceivers,
+    MovingLoopGroundFEMReceivers,
+)
+
+from simpeg_drivers import assets_path
+from simpeg_drivers.params import BaseForwardData, BaseInversionData, EMDataMixin
 
-from simpeg_drivers.params import InversionBaseParams
 
-from .constants import (
-    default_ui_json,
-    forward_defaults,
-    inversion_defaults,
-    validations,
+Receivers: TypeAlias = (
+    MovingLoopGroundFEMReceivers | LargeLoopGroundFEMReceivers | AirborneFEMReceivers
 )
 
 
-class FrequencyDomainElectromagneticsParams(InversionBaseParams):
+class FrequencyDomainElectromagneticsForwardParams(EMDataMixin, BaseForwardData):
```

Comment:
```suggestion
class FrequencyDomainElectromagneticsForwardData(EMDataMixin, BaseForwardData):
```
Can we rename them all so we make the distinction with old Params class? Would also be more inline with the base class. You can just use pycharm right-click `refactor/rename` to do it for you.

Reply:
I'm not sure `Data` is a good choice when it has such a specific meaning within simpeg-drivers.  Imagine in the next step we want to collect all the parameters pertaining to the data into a nested class, are we going to call this `DataData`?